starter/script.js

```diff
@@ -1 +1,93 @@
-// put code here!
+var side = prompt ("Your shirt is atrocious! I'm taking your dog! You'll have to solve my labyrinth. You have 13 mins. Pick your path! Left or Right! ");
+
+if (side == "left") {
+  alert("Boog of Eternal Strentch");
+
+  var boog_direction = prompt ("Oh! I see you're in the Boog of Eternal Stentch. Which way do you want to go. Left, or right. Your choice.");
+  var side;
+      if (side == "left") {
+          alert("My Rock friends help me across!I can go into the Forest.");
+      }
+      else (side == "right");{
+          alert("Oh No! I fell into the Bog of Eternal Stentch!");
+      }
+      var forest_direction = prompt ("I didn't think you'd get this far! You look hungry. Here have a peach?");
+         var eat;
+             if (eat == "Okay") {
```

Comment:
Next time, I recommend you take small steps and try it out along the way.  You have 93 lines of code, which break at line 8.